src/server/socketHandlers/scRelaySubscribeHandler.js

```diff
@@ -3,18 +3,35 @@ import {forAwaitEach} from 'iterall';
 import Schema from 'server/graphql/rootSchema';
 import handleGraphQLResult from 'server/utils/handleGraphQLResult';
 
+const trySubscribe = async (body, socket) => {
+  const {opId, query, variables} = body;
+  const authToken = socket.getAuthToken();
+  const context = {authToken, socketId: socket.id};
+  const document = parse(query);
+  const responseChannel = `gqlData.${opId}`;
+  try {
+    const result = await subscribe(Schema, document, {}, context, variables);
```

Comment:
Here's the promise that `subscribe` now returns. If it fails, it'll return an ExecutionResult (an object with `errors`). If the promise fails (as I'm doing in organizationUpdated) then we have to catch it below.

Reply:
This is cool!